pandas/core/frame.py

```diff
@@ -443,6 +451,43 @@
 1   foo      8
 2   bar      7
 3   bar      8
+
+Merge dataframes df1 and df2, where column df1.timestep falls within the range
+of df2.timestart to df2.timeend, using `condition`.
+
+>>> df1 = pd.DataFrame({'timestep': range(5)})
+>>> df2 = pd.DataFrame({'mood': ['happy', 'jolly', 'joy', 'cloud9'],
+...                     'timestart': [0, 2, 2, 3],
+...                     'timeend': [1, 3, 4, 4]})
+>>> df1
+   timestep
+0         0
+1         1
+2         2
+3         3
+4         4
+>>> df2
```

Comment:
use a blank line between things to make it easier to read

dont' call them df1, df2 instead use left,right (i know the other examples are like that but its too confusing). you can open an issue to fix that if you would like (or a PR even better)

Reply:
renamed and added the blank lines (added blank line in the other examples within this docstring as well)